src/util/config/config.cpp

```diff
@@ -359,19 +359,23 @@ namespace dxvk {
       { "d3d11.dcSingleUseMode",            "False" },
     }} },
     /* Rockstar Games Launcher                    */
-    { R"(\\Rockstar Games\\Launcher\\Launcher\.exe$)", {{
+    { R"(\\Rockstar Games\\Launcher\.exe$)", {{
       { "dxvk.maxChunkSize",                "1"   },
     }} },
     /* Rockstar Social Club                       */
-    { R"(\\Rockstar Games\\Social Club\\SocialClubHelper\.exe$)", {{
+    { R"(\\Rockstar Games\\SocialClub\\SocialClubHelper\.exe$)", {{
       { "dxvk.maxChunkSize",                "1"   },
     }} },
     /* EA Desktop App                             */
     { R"(\\EADesktop\.exe$)", {{
       { "dxvk.maxChunkSize",                "1"   },
     }} },
     /* Origin app (legacy EA Desktop)             */
-    { R"(\\Origin\.exe$)", {{
+    { R"(\\Origin\\OriginWebHelperService\.exe$)", {{
```

Comment:
Did you mean to remove the Origin exe itself?

Reply:
`R"(\\(Origin|OriginWebHelperService)\.exe$)"` or `R"(\\Origin(WebHelperService)?\.exe$)"` would be if you want both exe's.
`R"(\\Origin\\OriginWebHelperService\.exe$)"` is looking for `OriginWebHelperService.exe` inside a folder called `Origin`

Reply:
Yes